extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -61,4 +61,21 @@ public static function onArticleSaveComplete(
 
 		return true;
 	}
+
+	/**
+	 * Add community page entry point to article page right rail module
+	 *
+	 * @param array $railModuleList
+	 * @return bool
+	 */
+	public static function onGetRailModuleList( Array &$railModuleList ) {
+		$context = RequestContext::getMain();
+
+		if ( ( $context->getUser()->isLoggedIn() && $context->getTitle()->inNamespace( NS_MAIN ) ) ||
+			   $context->getTitle()->isSpecial( 'WikiActivity' ) ) {
```

Comment:
The acceptance criteria say:

On article pages for logged-in users, the CP entry point should be located at the very top of the Right Rail.
On Special:WikiActivity for everyone, the CP entry point should appear at the very top of the right rail

So I take this to mean that it should show regardless on WikiActivity.
(this means that the issue title is misleading)